### ISSUE TITLE
diesel migration redo -a broken

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/PermaplanT#674]
### Steps to Reproduce the Problem

run diesel migration redo -a

### Expected Result

Rolling back and redo of all the migrations

### Actual Result

with the database filled:

```
Rolling back migration 2023-07-11-095000_map_cascading_deletes
Rolling back migration 2023-07-03-165000_heatmap
Rolling back migration 2023-06-24-125250_planting_timeline_data
Rolling back migration 2023-06-20-110200_plants_relations
Rolling back migration 2023-06-13-102759_seed_owner
Rolling back migration 2023-06-12-110743_map_owner_uuid
Failed to run 2023-06-12-110743_map_owner_uuid with: Spalte »owner_id« von Relation »maps« enthält NULL-Werte
```

with an empty database:
see  #596

### PermaplanT version

master

### Operating System

Linux

### Browser

NA

**Comment by ThorbenStaufer** [comment]
Happens because a new "NOT NULL" field is added while there are already maps in the database. We'd either have to provide a default value or completely reset the database before redoing migrations. 
#644 should already fix this but it could pop up any time a new "NOT NULL" field is introduced to an already existing table.

**Comment by markus2330** [comment]
Thx for the explanation, I think it is fixed now due to #644.
